compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/res/PainterResources.desktop.kt

```diff
@@ -27,7 +27,8 @@ import org.xml.sax.InputSource
 
 /**
  * Load a [Painter] from an resource stored in resources for the application and decode
```

Comment:
Maybe write "stored in resources for the application (src/main/resources for jvm-only project)"?